instructions.ts

```diff
@@ -126,7 +126,8 @@ export default async function instructions(
   /**
    * Create the mail contracts file
    */
-  const contractsPath = app.makePath('contracts/mail.ts')
+  const contractsDir = app.directoriesMap.get('contracts') || 'contracts'
```

Comment:
Can we use `path.join` instead?

```suggestion
  const contractsDirectory = app.directoriesMap.get('contracts') || 'contracts'
  const contractPath = join(contractsDirectory, 'mail.ts')
```